gemini/use-cases/retrieval-augmented-generation/qna_using_query_routing/utils/qna_using_query_routing_utils.py

```diff
@@ -0,0 +1,163 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Utility file with more generic fuctions"""
+
+import configparser
+from datetime import datetime
+
+# Utils
+import os
+from typing import List
+
+from google.cloud import aiplatform
+from langchain.document_loaders import TextLoader, UnstructuredPDFLoader
+from langchain.text_splitter import CharacterTextSplitter
+from langchain_core.documents.base import Document
+
+config_file = "config.ini"
+config = configparser.ConfigParser()
+config.read(config_file)
+
+
+def get_deployed_index_id(
+    me_index_name: str, location: str
```

Comment:
```suggestion
    index_display_name: str, location: str
```

This is more clear about which name is being used.

Reply:
updated the parameter name to `index_display_name`.